api/common.js

```diff
@@ -413,6 +414,87 @@ exports.load_github_detail = function(service_name, cb) {
     }).catch(cb)
 }
 
+
+exports.generateQuery = function(str){
+    let result=stopwords.cleanText(str).split(' ').filter(e => String(e).trim());
+    if(result.length >0) {
+        result = result.map((word) => `W='${word}'`).join(',');
+        return "OR("+result+")"
+    }
+    return -1;
+}
+
+exports.updateProjectMag = function(project,cb){
+    if(!config.mag) return console.log("no mag config");
+    let headers = {
+        'Ocp-Apim-Subscription-Key': config.mag.subscriptionKey,
+        'User-Agent': 'brainlife',
+    }
+    const query = exports.generateQuery(project.name+" "+project.desc);
+    if(query === -1)
+        return cb();
+    let urlMag = "https://api.labs.cognitive.microsoft.com/academic/v1.0/evaluate?expr="+query+"&model=latest&count=10&offset=0&attributes=Id,AA.AfId,AA.AfN,AA.AuId,AA.AuN,AA.DAuN,CC,CN,D,Ti,F.FId,F.FN,Y,VFN,DOI,IA"
```

Comment:
This line is too long, and ugly. Please replace it by using `axios.get(url, {params: ....})` format.